sktime/transformations/panel/derivative.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+"""Derivative transformer."""
+import numpy as np
+
+from sktime.distances._ddtw import _average_of_slope
+from sktime.transformations.base import BaseTransformer
+
+
+class DerivativeTransformer(BaseTransformer):
```

Comment:
I would recommend not to give this name to the transformer - derivative has a common meaning in the time series domain, which is sometimes used interchangeably with the discrete version of it indicated by the `d` in ARIMA.

The formula for that `d`-erivative (or `d`-ifferences) is different from the funny "average of slope" used here, the coefficients are (1, -2, 1) and not (0.25, 0.5, -0.75).